addons/addon-base-raas/packages/base-raas-services/lib/aws-accounts/aws-cfn-service.js

```diff
@@ -27,6 +28,60 @@ const { processInBatches } = require('@aws-ee/base-services/lib/helpers/utils');
 
 const settingKeys = {
   awsRegion: 'awsRegion',
+  envBootstrapBucket: 'envBootstrapBucketName',
+  apiHandlerRoleArn: 'apiHandlerArn',
+  workflowLoopRunnerRoleArn: 'workflowRoleArn',
+  swbMainAccount: 'mainAcct',
+  stage: 'envName',
+};
+
+const getCreateStackUrl = (cfnTemplateInfo, createParams) => {
+  // see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stacks-quick-create-links.html
+  const { name, region, signedUrl } = cfnTemplateInfo;
+  const { apiHandlerRoleArn, workflowLoopRunnerRoleArn, mainAcct, externalId, namespace } = createParams;
+  const url = [
+    `https://console.aws.amazon.com/cloudformation/home?region=${region}#/stacks/create/review/`,
+    `?templateURL=${encodeURIComponent(signedUrl)}`,
+    `&stackName=${name}`,
+    `&param_Namespace=${namespace}`,
+    `&param_CentralAccountId=${mainAcct}`,
+    `&param_ExternalId=${externalId}`,
+    `&param_ApiHandlerArn=${apiHandlerRoleArn}`,
+    `&param_WorkflowRoleArn=${workflowLoopRunnerRoleArn}`,
+  ].join('');
+
+  // This one takes us directly to the review stage but will require that we access the cloudformation console first
+  // const url = [
+  //   `https://console.aws.amazon.com/cloudformation/home?region=${region}#/stacks/create/review/`,
+  //   `?templateURL=${encodeURIComponent(signedUrl)}`,
+  //   `&stackName=${name}`,
+  // ].join('');
+
+  // This takes us to the create new page:
+  // `https://console.aws.amazon.com/cloudformation/home?region=${region}#/stacks/new`,
+  // Note that this doesn't populate parameters correctly
+
+  return url;
+};
+
+const getUpdateStackUrl = cfnTemplateInfo => {
+  const { stackId, region, signedUrl } = cfnTemplateInfo;
```

Comment:
Can we add parameters update as well? There might be a situation where parameter update is needed.

Reply:
I'm not sure what you mean--do you mean to explicitly put the parameters into the URL like it is in the getCreateStackUrl method?

Reply:
Yes, if one of the parameter value changed, a stack update should catch that as well.

Reply:
It doesn't look like this is possible unfortunately, the functionality to add parameters into the URL is only included as part of [quick-creating stack URLs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stacks-quick-create-links.html), and there doesn't seem to be an equivalent for updating a stack. The parameters shouldn't need to be changed often though--in the event they are it's much easier to update from the AWS management console than it is to update the entire stack's template.